docs/sources/clients/promtail/_index.md

```diff
@@ -91,8 +91,8 @@ This endpoint returns 200 when Promtail is up and running, and there's at least
 
 ### `GET /metrics`
 
-This endpoint returns Promtail metrics for Prometheus. See
-"[Operations > Observability](../../operations/observability/)" to get a list
+This endpoint returns Promtail metrics for Prometheus. Refer to
```

Comment:
Glad to find this laternative :slightly_smiling_face: . I always wrote `See` but it didn't feel right.